velox/functions/prestosql/tests/DecimalArithmeticTest.cpp

```diff
@@ -174,3 +174,43 @@ TEST_F(DecimalArithmeticTest, decimalSubTest) {
       "minus(c0, c1)",
       {shortWithNullsA, shortWithNullsB});
 }
+
+TEST_F(DecimalArithmeticTest, decimalMulTest) {
+  auto shortFlat = makeShortDecimalFlatVector({1000, 2000}, DECIMAL(17, 3));
+  auto expectedLongFlat =
```

Comment:
Can we inline some of these variables that are only used once? It is easier to read that way.

Reply:
Same for subtraction and addition tests also.